R/summarize_company_production.R

```diff
@@ -29,7 +29,7 @@
 summarize_company_production <- function(data,
                                          ...,
                                          use_credit_limit = FALSE) {
-  crucial <- c("name_ald", "scenario", "tmsr", "smsp")
+  crucial <- c("name_ald", "scenario", "tmsr", "smsp", "region")
```

Comment:
Maybe extract `"scenario", "tmsr", "smsp", "region"`? I see it repeated in `summarize_portfolio_production()`. If they are likely to change together, then having a single representation of this pattern makes sense.